WordPress/Classes/Services/EditorSettings.swift

```diff
@@ -67,4 +84,42 @@ class EditorSettings: NSObject {
             database.set(newValue, forKey: nativeEditorEnabledKey)
         }
     }
+
+    // We can't return a type that's both a PostEditor and a UIViewController, so using
+    // a configure block as a hack.
+    // In Swift 4, we'll be able to do `instantiateEditor() -> UIViewController & PostEditor`,
+    // and then let the caller configure the editor.
+    func instantiatePostEditor(post: AbstractPost, configure: (PostEditor, UIViewController) -> Void) -> UIViewController {
+        switch editor {
+        case .aztec:
+            let vc = AztecPostViewController(post: post)
+            configure(vc, vc)
```

Comment:
`configure(vc, vc)` and `return vc` are both duplicated three times here, can that become just one call each?